fmralign/template_alignment.py

```diff
@@ -265,7 +266,7 @@ def __init__(
         alignment_method: string
             Algorithm used to perform alignment between X_i and Y_i :
             * either 'identity', 'scaled_orthogonal', 'optimal_transport',
-            'ridge_cv', 'permutation', 'diagonal'
+            'ridge_cv', 'permutation', 'diagonal', 'pcha', 'slha'
```

Comment:
I guess that these names are not standard ---- I'm not familiar with  them. Can you provide a more explicit naming ?

Reply:
I guess `searchlight_hyperalignment` and `parcel_hyperalignment`.

Reply:
yes that is it, i wanted to have short names for different methods